packages/harpguru-core/src/components/menu-open-button/hooks/use-tap-animation/use-tap-animation.ts

```diff
@@ -0,0 +1,44 @@
+import { useTimingTransition } from 'react-native-redash'
+import { Node, Easing, interpolate } from 'react-native-reanimated'
+import { State } from 'react-native-gesture-handler'
+import type { TapGestureHandlerStateChangeEvent } from 'react-native-gesture-handler'
+import React from 'react'
+
+import { usePrevious } from '../../../../hooks'
+
+type TapEventHandler = (arg0: TapGestureHandlerStateChangeEvent) => void
+
+export const useTapAnimation = (
+  openCloseMenu: () => void
+): [Node<number>, TapEventHandler] => {
+  const [isTapped, setIsTapped] = React.useState(false)
```

Comment:
Again,  would be good if we could do this in a shared hook with everything else that handles taps.

Reply:
Done